src/app/pages/form/reactive-forms/reactive-forms.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="tds-u-flex tds-u-flex-dir-col tds-u-flex-gap-3">
+<div class="tds-u-flex tds-u-flex-dir-col tds-u-gap3">
   <h3>Reactive forms</h3>
   <p>This form is built using Angular Reactive forms. It needs some manual work to make it fully function with our components</p>
     <form (submit)="submitForm($event)">
```

Comment:
It might be a larger one to improve as we listen to URL changes and once you submit form, url changes to it. I have preventedDefault on it so we just console.log it for now. Thanks for finding this one!